recipes/libtool/all/test_package/static-in-shared/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(static_dependency C)
+
+add_library(static_lib STATIC
+    static_lib.c
+)
```

Comment:
Have you tested on Unix? I guess that PIC should be enabled if this lib is linked into a shared lib.

Reply:
You're right. But it works without.
I have no clue why though..

```
[ 50%] Building C object CMakeFiles/static_lib.dir/static_lib.c.o
/usr/bin/cc   -O2 -DNDEBUG -o CMakeFiles/static_lib.dir/static_lib.c.o -c /home/maarten/programming/conan-center-index/recipes/libtool/all/test_package/build/0bb5c6b789c31ce1a9e22c28a450347b2c15ae87/static-in-shared/static_lib.c
```